### ISSUE TITLE
No `ICONST` opcode in IT pack?

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/opeo-maven-plugin#25]
Consider this java class:
```java
package com.example;
public class B {
    private A a;
    public B(A a) { this.a = a; }
    public int bar() { return this.a.foo() + 2; }
}
```

After `jeo-maven-plugin` method `bar` looks like this:
```
  [] > j$bar
    1 > access
    "()I" > descriptor
    "" > signature
    * > exceptions
    seq > @
      tuple
        label
          "00e4cf3f-4bc4-49fe-9ed1-99d824aee21c"
        opcode > ALOAD-7
          25
          0
        opcode > GETFIELD-8
          180
          "com/example/B"
          "a"
          "Lcom/example/A;"
        opcode > INVOKEVIRTUAL-9
          182
          "com/example/A"
          "foo"
          "()I"
        opcode > ICONST_2-A
          5
        opcode > IADD-B
          96
        opcode > IRETURN-C
          172
        label
          "319d857e-35fd-4c87-bf06-a16b147474f3"
    tuple > trycatchblocks
```
In your [pack](https://github.com/objectionary/opeo-maven-plugin/blob/master/src/test/resources/packs/simple_objects.yaml) for IT it becomes to look a bit different:
```
  [] > j$bar
    1 > access
    "()I" > descriptor
    "" > signature
    * > exceptions
    seq > @
      tuple
        opcode > ALOAD-13
          25
          0
        opcode > GETFIELD-14
          180
          "com/example/B"
          "a"
          "Lcom/example/A;"
        "this"
        .foo
        opcode > IADD-15
          96
        opcode > IRETURN-16
          172
    tuple > trycatchblocks
```

1. A'm I right that `"this".foo` is the result of decompilation?
2. Where did `ICONST` opcode go?

**Comment by volodya-lombrozo** [comment]
@maxonfjvipon Seems like it is a bug. I will try to solve it soon

**Comment by volodya-lombrozo** [comment]
@maxonfjvipon I fixed this problem in the '0.0.4' version. Could you try again, please?

**Comment by volodya-lombrozo** [comment]
@maxonfjvipon Since there is no activity anymore, I close the issue. Please, let me know if you have similar problem.
